sample-operators/webpage/src/main/java/io/javaoperatorsdk/operator/sample/WebPageStandaloneDependentsReconciler.java

```diff
@@ -37,56 +37,50 @@
   private static final Logger log =
       LoggerFactory.getLogger(WebPageStandaloneDependentsReconciler.class);
 
-  private KubernetesDependentResource<ConfigMap, WebPage> configMapDR;
```

Comment:
I think this is an unnecessary simplification I actually plan to extend this sample with some `if` or other workflow enahncements, that is not possible with managed dependent resources.

Basically that is the point that we can hold a simple reference to the dependent resource, that can be easily accessed. Putting it into a map and accessing it via String, is just unnecessary complicaiton. 

See also other PR pl with defaul naming.

Reply:
Sure but the current example only shows code duplication which isn't really something we want people to replicate.

Reply:
And calling reconcile on each dependent and create the dependents each exactly the same way… Basically, as it stands, this sample isn't really interesting (apart from showing that people should probably use managed dependents for this kind of use case 😄) so I'd just leave it as it is here and then we can make more changes to make it more interesting in other PRs.

Reply:
Yes, right, will come up with something to improve it.

Btw otherwise look good to me. Happy to merge! Pls do. ( Just don't know how to approve from mobile app :) )